rpm/0003-Revert-Async-api-for-certificate-validation.patch

```diff
@@ -0,0 +1,1171 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Anton Thomasson <antonthomasson@gmail.com>
+Date: Sat, 29 Jun 2024 22:09:09 +0200
+Subject: [PATCH] Revert "Async api for certificate validation"
```

Comment:
As the one who did this upstream, given Sailfish' requirements I think it is an okay solution to revert this.

Also, internally, the async certificate validation is only fully implemented in the in-sailfish disabled gpgme backend.

It is also only newest Okular that so far uses it, and that Okular is also based on Qt6.